.github/workflows/distribute.yml

```diff
@@ -0,0 +1,55 @@
+name: Distribute to PyPI
+
+on:
+  push:
+    tags:
+      - v*
```

Comment:
What about accidental tags to branches different from main? should we just release if the tag is on main?

Reply:
In general, I don't really use tags outside of version numbers, so I'm not terribly worried about other tags.

We could be more specific with the pattern like `v*.*.*`, but I don't think there's a way to do "and" logic with branches and tags because they all separate refs (github actions logic is generally "or").